content/src/main/content/jcr_root/apps/apm/components/reference/reference.html

```diff
@@ -0,0 +1,136 @@
+<!--/*
+* ========================LICENSE_START=================================
+* AEM Permission Management
+* %%
+* Copyright (C) 2013 - 2016 Cognifide Limited
+* %%
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+* =========================LICENSE_END==================================
+*/-->
+<div style="margin: 3%">
+  <sly data-sly-use.model="com.cognifide.cq.cqsm.core.models.ReferencesModel">
+    <h3>Available commands</h3>
+    <table is="coral-table">
+      <thead is="coral-table-head">
+      <tr is="coral-table-row">
+        <th style="width: 35%;" is="coral-table-headercell">Command syntax</th>
+        <th style="width: 20%;" is="coral-table-headercell">Arguments</th>
+        <th style="width: 45%;" is="coral-table-headercell">Description</th>
+      </tr>
+      </thead>
+      <tbody is="coral-table-body">
+      <sly data-sly-repeat.map="${model.references}">
+        <sly data-sly-repeat="${map}">

```

Comment:
Why do you put a second repeat here? I think that you may simlify this markup:
```
<sly data-sly-repeat.map="${model.references}">
  <tr is="coral-table-row">
    <td is="coral-table-cell">${map["commands"] @ join='<br> ', context='html'}</td>
    <td is="coral-table-cell">${map["args"] @ join=', '}</td>
    <td is="coral-table-cell">${map["reference"]}</td>
  </tr>
</sly>

Reply:
Fixed